popup/js/search/fuzzySearch.js

```diff
@@ -34,48 +34,46 @@ export async function fuzzySearch(searchMode, searchTerm) {
 function fuzzySearchWithScoring(searchTerm, data) {
   /** Search results */
   const results = []
+  const haystack = data.map((el) => {
+    return el.searchString
+  })
 
-  // Score with fuzzysort is difficult to handle.
-  // We assume that the worst score we allow is -500.000
-  // and normalize score between this and 0
-  const scoreNormalizationFactor = -500000
-
-  const searchResults = fuzzysort.go(searchTerm, data, {
-    key: 'searchString',
-    limit: ext.opts.searchMaxResults,
-    // subtracting 1, otherwise the library will get confused by -0
-    threshold: ext.opts.searchFuzzyness * scoreNormalizationFactor - 1,
+  let uf = new uFuzzy({
```

Comment:
this to good to cache as well, but i guess not possible either if haystack caching isnt possible :(

Reply:
yes, I'm afraid so.